poradnia/advicer/filters.py

```diff
@@ -17,6 +19,10 @@ class AdviceFilter(CrispyFilterMixin, django_filters.FilterSet):
     advicer = UserChoiceFilter(label=_("Advicer"))
     created_by = UserChoiceFilter(label=_("Created by"))
     created_on = django_filters.DateRangeFilter(label=_("Created on"))
+    community = AreaFilter(
+        label=_("Community"),
+        widget=autocomplete.ModelSelect2(url='teryt:community-autocomplete')
```

Comment:
Zastanawiam się czy nie warto byłoby wystawić odrębny url (np. `teryt:community-full-path-autocomplete` który zwracałby nie tyle nazwę gminy, co całą ścieżkę `województwo/powiat/gmina`, tj. zamiast `Gmina Kraków` -> `Małopolskie/Krakowski/Kraków` (lub coś w tym stylu). W obecnej sytuacji nie ma możliwości odróżnienia różnych gmin o tej samej nazwie, choć jest to (o ile wiem) częsty przypadek.

Reply:
@ad-m wystawiłem PR #739 który wyświetla pełne ścieżki tam gdzie używamy nazw gmin.

Reply:
Dzięki. Zatem jak przejdą testy to ten PR włączę.